### PR TITLE
Add new benchmark suites

### PR DESCRIPTION
**Pull request by turbo0628**

### CODE REVIEW

File: scripts/prepare_conda_envs_for_taichi_releases.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ENV_PREFIX="taichi_perfmon_releases"
+TI_VERSIONS=(1.0.0 1.0.1 1.0.2 1.0.3 1.0.4 1.1.0 1.1.2 1.1.3 1.2.0 1.2.1 1.2.2 1.3.0 master)
+PY_VERSION="3.9"
+CONDA_ENVS=`conda env list`
+for TI_VER in ${TI_VERSIONS[@]}
+do
+ENV_NAME="${ENV_PREFIX}_${TI_VER}"
+ENV_AVAIL=`echo ${CONDA_ENVS}|grep ${ENV_NAME}`
+if [ -z "$ENV_AVAIL" ]; then
+echo "[Taicho] Taichi release environment ${ENV_NAME} not available yet. Create it!"
```
Comment (feisuzhu) [inline scripts/prepare_conda_envs_for_taichi_releases.sh:12]: `Taicho` typo

File: suites/stencil/taichi/stencil.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            for I in ti.grouped(x):
+                if (I[0] == 0) or (I[1] == 0) or (I[0] == N - 1) or (I[1] == N - 1):
+                    x[I] = y[I]
+                else:
+                    p = 0.0
+                    for stencil in ti.static(neighbours):
+                        p += y[I + stencil]
+                    x[I] = p * 0.25
+
+        self.run_iter = lambda: jacobi_step()
```
Comment (feisuzhu) [inline suites/stencil/taichi/stencil.py:54]: `self.run_iter = jacobi_step` is sufficient.

**Review by feisuzhu — APPROVED** [review]
LGTM

File: scripts/prepare_conda_envs_for_taichi_releases.sh (hunk 1 above)
Comment (turbo0628) [reply scripts/prepare_conda_envs_for_taichi_releases.sh:12]: Thanks!

File: suites/stencil/taichi/stencil.py (hunk 2 above)
Comment (turbo0628) [reply suites/stencil/taichi/stencil.py:54]: Fixed!

**Pull request merged**